### DISCUSSION TITLE
Скачивается глава без страниц

### DISCUSSION THREAD

**Discussion by Meloni4** [lirix360/ReadmangaGrabber#63] [Q&A]
Пытался скачать мангу: https://readmanga.live/ohotnik_x_ohotnik__A5327#tran_110348 и скачивается пустая папка в любом из переводов, а к примеру с этой мангой: https://mintmanga.live/chelovek_benzopila__A5327 всё скачивается нормально в любом из переводов.

**Comment by lirix360** [comment]
> Пытался скачать мангу: https://readmanga.live/ohotnik_x_ohotnik__A5327#tran_110348 и скачивается пустая папка в любом из переводов, а к примеру с этой мангой: https://mintmanga.live/chelovek_benzopila__A5327 всё скачивается нормально в любом из переводов.

Уберите "#tran_110348" из адреса.
